tests/general/SpaceConsistencyBearTest.py

```diff
@@ -26,26 +26,26 @@ def test_defaults(self):
         self.section.append(Setting('use_spaces', 'true'))
 
         self.check_validity(self.uut, ['    t'])
-        self.check_validity(self.uut, ['\tt'], valid=False)
-        self.check_validity(self.uut, ['t \n'], valid=False)
-        self.check_validity(self.uut, ['t'],
-                            force_linebreaks=False, valid=False)
+        self.check_invalidity(self.uut, ['\tt'])
+        self.check_invalidity(self.uut, ['t \n'])
+        self.check_invalidity(self.uut, ['t'],
+                            force_linebreaks=False)
```

Comment:
@prachi1210 This is just an issue of improper indentation. Add proper indentation to line 32, currently it's under indented. You could apply the following patch to fix it directly.